lib/resque/plugins/retry/hooks.rb

```diff
@@ -0,0 +1,62 @@
+module Resque
+  module Plugins
+    module Retry
+      module Hooks
+        # Returns a sorted list of hook methods, according to the hook prefix.
+        #
+        # @param hook_prefix [String]
+        #
+        # @returns [Array<Symbol>]
+        #
+        # @api private
+        def get_hook_names(hook_prefix)
```

Comment:
I was just copying what the main Resque project does. They return the hooks in sorted order so that they get invoked in sorted order. See: https://github.com/resque/resque/blob/master/lib%2Fresque%2Fplugin.rb#L50